Gemfile

```diff
@@ -6,11 +6,6 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 branch = ENV.fetch('SOLIDUS_BRANCH', 'master')
 gem 'solidus', github: 'solidusio/solidus', branch: branch
 
-# Needed to help Bundler figure out how to resolve dependencies,
-# otherwise it takes forever to resolve them.
-# See https://github.com/bundler/bundler/issues/6677
-gem 'rails', '>0.a'
-
```

Comment:
Why is this removed?

Reply:
No longer required. There used to be an issue with bundler versions below 2.2 I believe which caused hanging when resolving dependencies for older versions of rails which this line of code rectified. The bug in bundler has been rectified so I was just removing it since I was working on the repo.